setup/main.tf

```diff
@@ -286,19 +286,40 @@ resource "aws_ecs_task_definition" "steampulse_dashboard_task_definition" {
   container_definitions = jsonencode([
     {
       name   = "steampulse_dashboard_ecr"
-      image  = "na"
+      image  = "129033205317.dkr.ecr.eu-west-2.amazonaws.com/steampulse_dashboard_ecr:latest"
       cpu    = 10
```

Comment:
I'm not super happy that I'm hardcoding the container address here, that's something I could improve, but at least it's working for now

Reply:
Would be good to move to vars